Formula/rmw.rb

```diff
@@ -0,0 +1,30 @@
+class Rmw < Formula
+  desc "Safe-remove utility for the command-line"
+  homepage "https://remove-to-waste.info/"
+  url "https://github.com/theimpossibleastronaut/rmw/releases/download/v0.7.09/rmw-0.7.09.tar.gz"
+  sha256 "d0a85944d03e7ec2a94b25d6d6ac92880fd0d3f63d90bb9ed56b16418fd41c69"
+  license "GPL-3.0-or-later"
+  head "https://github.com/theimpossibleastronaut/rmw.git"
+
+  depends_on "ncurses"
+
+  def install
+    ENV["PKG_CONFIG_PATH"] = "/usr/local/opt/ncurses/lib/pkgconfig"
```

Comment:
Can we build without this? You don't actually have a `pkg-config` dependency, so I'm pretty sure this is not used. We also shouldn't be hardcoding a `/usr/local` prefix here.

Reply:
> Can we build without this? You don't actually have a `pkg-config` dependency, so I'm pretty sure this is not used. We also shouldn't be hardcoding a `/usr/local` prefix here.

I changed it to `ENV.append_path "PKG_CONFIG_PATH", Formula["ncurses"].opt_lib/"pkgconfig"`
> 
> `brew` also automatically specifies `PKG_CONFIG_PATH` correctly based on dependencies.

I've always needed it when building this program on Travis, using brew to install ncurses. And I just set it up on GitHub actions. First time I tried without using PKG_CONFIG_PATH, it failed. I added it, and it worked.

Reply:
As mentioned in my other comment, your Travis build runs outside `brew`, so of course it doesn't benefit from `brew` setting the env variables correctly. We definitely don't need this. Please remove it.